dockershrink/cli.py

```diff
@@ -4,41 +4,100 @@
 import os
 import traceback
 from pathlib import Path
-
-import openai
```

Comment:
removing this is causing errors because we're using `openai.APIStatusError` and APIError.
So its better to keep both `import openai` and `from openai import openai`.